### PR TITLE
Add script for verifying API responses won't change

### PR DESCRIPTION
**Pull request by hqpho**
These validate that all endpoints continue to work with routing through a new load balancer.

### CODE REVIEW

File: tools/migrationtesting/compare_responses.py
Comment (beets) [inline tools/migrationtesting/compare_responses.py:1]: super nit: rename migrationtesting to migration_testing

File: tools/migrationtesting/compare_responses.py [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+    # 405
+    ("/v1/recon/resolve/id", ["GET"], {}),
+    # 415
+    ("/query?name=example.com&type=A", ["GET"], {}),
+    # 500
+    ("/v1/recon/resolve/id", ["POST"], {}),
+]
+
+
+class bcolors:
```
Comment (beets) [inline tools/migrationtesting/compare_responses.py:394]: fancy!

**Review by beets — APPROVED** [review]
very nice - thanks for doing the detailed diffs for the migration

File: tools/migrationtesting/compare_responses.py
Comment (hqpho) [reply tools/migrationtesting/compare_responses.py:1]: Done.

**Pull request merged**